### ISSUE TITLE
[Bug]: Can't Open Publii - "Cannot find your sites folder"

### ISSUE THREAD

**Issue by sunrays2993** [GetPublii/Publii#1659]
### Operating system

Winods 10 x64

### Publii version

0.46.1

### Post editor

None

### Bug description

When I attempt to open Publii I get the error of "Publii cannot find your sites folder". Please check if the directory C:\Users\James\Documents\Publii\sites\ exists. Publii will not open at all, not even give me an option to edit any settings.

The folder does exist and worked the last time I opened Publii, 2 months ago.

The version I originally had installed was 0.45.2 and received this error. I updated to 0.46.1 but the error persists. Uninstalled even with a specific uninstall software (BC Uninstaller) which resulted in deletion of leftover files & registry entries, but still not working even when 0.46.1 is installed.

![image](https://github.com/user-attachments/assets/57bae593-f282-485f-a13f-da38b33745d2)

### Relevant log output

No loggs that I am aware of

**Comment by sunrays2993** [comment]
Forgot to add that Publii will not open at all, not even give me an option to edit any settings.

**Comment by dziudek** [comment]
Hi,

The same problem exists if you install v.0.45.2? 

https://cdn.getpublii.com/Publii-0.45.2.exe

**Comment by sunrays2993** [comment]
Hello @dziudek 
I did install 0.45.2 but the problem still occured.

I have solved it. The problem was relating to the website data inide of the folder 'C:\Users\James\Documents\Publii\'.

I renamed the 'Documents\Publii' folder to 'Documents\Publii(OLD)', and opened the Publii program OK.

I then selected restore from backup and chose the back inside of the 'Documents\Publii(OLD)' folder & it created  a new 'Documents\Publii' folder, where the data is now stored.

All working OK.
